### DISCUSSION TITLE
Integration Object Utility

### DISCUSSION THREAD

**Discussion by pk-pankaj** [breezefront/community#239] [1. Help] [answered]
Hi @vovayatsyuk 

I am trying to reuse Luma files while integration and have to make utility for object but that is not available in the docs, so can you please help on this?

used in define

```
define([
    'My_Module/js/reload',
], function (module) {
```

and my file like this

```
define([
    'jquery'
], function ($) {
    'use strict';

    return {
        
    };
});
```

Thanks,
Pankaj

**Comment by vovayatsyuk** [comment, marked as answer]
Hi,

You need to add `component: 'My_Module/js/reload'` line to your object:

```js
define([
    'jquery'
], function ($) {
    'use strict';

    return {
        component: 'My_Module/js/reload',
        ...
    };
});
```

**Reply by pk-pankaj** [reply]
It works, thanks
